test.js.md

```diff
@@ -9,52 +9,60 @@ Generated by [AVA](https://ava.li).
 > Snapshot 1
 
     {
-      authentication: {
-        password: 'password',
-        username: 'username',
-      },
-      clickElement: 'button',
-      cookie: 'id=unicorn; Expires=Wed, 21 Oct 2018 07:28:00 GMT;',
-      cookies: [
-        'id=unicorn; Expires=Wed, 21 Oct 2018 07:28:00 GMT;',
-      ],
+      'authenticationusername:passwor': true,
+      clickElementbutto: true,
+      cookieid: 'unicorn; Expires=Wed, 21 Oct 2018 07:28:00 GMT',
+      cookies: [],
       debug: false,
       delay: 10,
-      element: '.main-content',
-      emulateDevice: 'iPhone X',
-      fullPage: false,
-      header: 'x-powered-by: capture-website-cli',
-      headers: {
-        'x-powered-by': 'capture-website-cli',
+      element: {
+        'main-conten': true,
```

Comment:
This is not correct

Reply:
Uhm, I just launched `ava -u`

Reply:
There's clearly something incorrect about your replacement thing: https://github.com/sindresorhus/capture-website-cli/pull/2/files#diff-1dd241c4cd3fd1dd89c570cee98b79ddR47

`main-conten` is missing a `t`, for example.

Reply:
Fixed.